libs/prime/restake/src/components/TokenInput.tsx

```diff
@@ -35,7 +35,7 @@ export type TokenInputProps = {
     BigintInputProps,
     'value' | 'decimals' | 'onChange' | 'isLoading' | 'isError'
   >;
-  tokenButtonProps?: Omit<TokenButtonProps, 'token'>;
+  tokenButtonProps?: Omit<TokenPickerProps, 'token'>;
```

Comment:
this isn't used in the component, maybe remove it

Reply:
This is just a side-effect from a rename so I paid no attention to content here.